### PR TITLE
created form component for edit.html.erb and new.html.erb

### PR DESCRIPTION
**Pull request by marshallekm**
I changed the edit.html.erb and new.html.erb and added form component instead.

styling pretty much unchanged

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 class Project < ApplicationRecord
   belongs_to :user
-  validates :project_name, presence: true, uniqueness: true
```
Comment (switchloops) [inline app/models/project.rb]: the validations are all commented out, un comment please

**Review by Reona001 — COMMENTED** [review]
Please uncomment the validations for the Project model.
Thanks!

**Review by switchloops — CHANGES_REQUESTED** [review]

File: app/views/projects/edit.html.erb [hunk 2]
```diff
@@ -1,13 +1,3 @@
...
-    <%= f.input :location %>
-    <%= f.input :start_date %>
-    <%= f.input :end_date %>
-    <%= f.input :reward_earned %>
-    <%= f.input :points_earned %>
-    <%= f.submit class: "btn btn-primary" %>
-  <% end %>
-</div>
\ No newline at end of file
+<%= render 'components/project-form'  %>
```
Comment (switchloops) [inline app/views/projects/edit.html.erb]: indent this

File: app/views/projects/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  <div class="form-container">
+    <div class="w-50 mx-auto">
+      <h2>Create your project </h2>
+    <%= render 'components/project-form'  %>
```
Comment (switchloops) [inline app/views/projects/new.html.erb]: indent this to same as h2

**Review by switchloops — CHANGES_REQUESTED** [review]

**Pull request merged**